caniusepython3/__init__.py

```diff
@@ -21,33 +21,7 @@
 
 import multiprocessing
 
-
 try:
     CPU_COUNT = max(2, multiprocessing.cpu_count())
 except NotImplementedError:  #pragma: no cover
     CPU_COUNT = 2
-
-
-def check(requirements_paths=[], metadata=[], projects=[]):
```

Comment:
Why was this and its tests removed? This is part of the documented API for the package.